qiita_db/study.py

```diff
@@ -166,6 +166,18 @@ def status(self):
             return qdb.util.infer_status(
                 qdb.sql_connection.TRN.execute_fetchindex())
 
+    @staticmethod
+    def all_data_types():
+        """Returns list of all the data types available in the system
+        Returns
+        -------
+        list of str
```

Comment:
Mind describing what the strings represent? I know its repetitive, but that's the suggested way.